HaikuPorter/Configuration.py

```diff
@@ -146,6 +146,15 @@ def isExecutable(path):
 		'optionAttribute': 'commandPackage',
 		'setAttribute': 'packageCommand',
 	},
+	'PACKAGE_COMPRESSION_LEVEL': {
+		'type': bytes,
```

Comment:
why not set it to type int instead?

Reply:
Because I'm dumb :-) (did a quick look for other types used, didn't saw int... went with what was already there).

Thanks for the note, will try it today. (same for the comment on Options.py below).